src/reducers/petitions.js

```diff
@@ -16,10 +17,15 @@ export default function petitions (state = initialState, action) {
       return Object.assign({}, state,
         action.petitions, {
           isLoading: false,
+          totalCount: action.petitions.total,
```

Comment:
you might remove this if we don't need to know it until inside the `Pagination` component? (which has its own helper to generate props)